app/controllers/user_targets_controller.rb

```diff
@@ -16,6 +16,15 @@ def create
     end
   end
 
+  def show
+    @target = UserTarget.find(params[:id])
```

Comment:
You can move this line to a private method and load the user in a `before_action`.
```
before_action :load_target, only: [:show, :destroy]
...
def show
end

def destroy
@target.destroy
end

private

def load_target
@target = UserTarget.find(params[:id])
end
```